saltyrtc/server/server.py

```diff
@@ -45,7 +58,10 @@
 
 
 @asyncio.coroutine
-def serve(ssl_context, key, paths=None, host=None, port=8765, loop=None):
+def serve(
+        ssl_context, key, paths=None, host=None, port=8765, loop=None,
+        event_callbacks: Dict[Event, List[Coroutine]] = None
```

Comment:
This is in line with [PEP 484 recommendations](https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values). PEP8 refers to PEP484 for syntax in this case.

The thing is that `event_callbacks: Dict[Event, List[Coroutine]]=None` looks weird, as if `=None` were part of the type.